packages/core-data/src/types/helpers.ts

```diff
@@ -0,0 +1,164 @@
+/**
+ * Internal dependencies
+ */
+import { WpTemplate } from './wp-template';
+import { WpTemplatePart } from './wp-template-part';
+import { EntityRecord } from './index';
+
+export interface AvatarUrls {
+	/**
+	 * Avatar URL with image size of 24 pixels.
+	 */
+	'24': string;
+	/**
+	 * Avatar URL with image size of 48 pixels.
+	 */
+	'48': string;
+	/**
+	 * Avatar URL with image size of 96 pixels.
+	 */
+	'96': string;
+}
+
+export type MediaType = 'image' | 'file';
+export type CommentStatus = 'open' | 'closed';
+export type PingStatus = 'open' | 'closed';
+export type PostStatus = 'publish' | 'future' | 'draft' | 'pending' | 'private';
+export type PostFormat =
+	| 'standard'
+	| 'aside'
+	| 'chat'
+	| 'gallery'
+	| 'link'
+	| 'image'
+	| 'quote'
+	| 'status'
+	| 'video'
+	| 'audio';
```

Comment:
we still have these types in common but now we also have the types defined in the related modules. I don't think we meant to have both. for example, when I was talking about comment status I was referring to this existing copy. the new one in `comment` is better, but maybe we don't need this here anymore then.

Reply:
Ah, we have two different types called `CommentStatus`. One relates to the ability to comment on something, e.g. post may have `comment_status = 'open'`, the other describes the status of a specific comment, like `approve` or `trash`. I renamed the former to `CommentingStatus` to indicate it's about the activity and not the comment instance.

Reply:
I double-checked all the other types and I don't think there are any duplicate declarations left.